index.html

```diff
@@ -49,6 +49,212 @@ <h1 class="pad-top-158 font-size-40 font-weight-800 line-height-52 slategray mar
             <li><a href="#" class="margin-left-16"><img src="/assets/angelist.svg" alt="angellist icon"></a></li>
         </ul>
     </section>
+
+    <div class="grid d-grid">
+        <article class="pad-24 margin-top-6em">
+            <div class="cards d-flex flex-column align-center background-white pad-16 border-radius-16 pad-bottom-30">
+                <div class="cards-img img background-no-repeat">
+                </div>
+                <div class="cards-text">
+                    <h4 class="cards-text-header bold font-size-32 line-height-44 black pad-t-8 slategray"> Tonic</h4>
+                    <ul class="d-flex line-height-16 font-weight-600 font-size-13 pad-top-1em">
+                        <li class="">CANOPY</li>
+                        <li class="box margin-left-16"></li>
+                        <li class="margin-left-16 grey">Back End Dev</li>
+                        <li class="box margin-left-16"></li>
+                        <li class="margin-left-16 grey">2015</li>
+                    </ul>
+                    <p class="text font-weight-400 font-size-16 line-height-24 slateblu pad-top-1em">A daily selection
+                        of
+                        privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="d-flex pad-top-12 pad-bottom-2em">
+                        <li
+                            class="font-size-12 font-weight-500 background-grey line-height-20 dodgerblue height-20 border-radius-7 width-53 pad-left-13">
+                            html</li>
+                        <li
+                            class="font-size-12 font-weight-500 background-grey line-height-20 dodgerblue margin-left-8 height-20 border-radius-7 width-45 pad-left-12">
+                            css
+                        </li>
+                        <li
+                            class="font-size-12 font-weight-500 background-grey line-height-20 dodgerblue margin-left-8 height-20 border-radius-7 width-65 pad-left-7">
+                            javascript</li>
+                    </ul>
+                    <button type="button"
```

Comment:
- There is no button interaction implemented on any of the buttons on the page. Please work on this. It is suggested by the requirement screenshot below:

    ![requirement](https://user-images.githubusercontent.com/45002654/125007313-9c64bb80-e068-11eb-8744-7341c9dd5b03.PNG)

Reply:
👍🏽